app/assets/javascripts/custom.js

```diff
@@ -27,6 +27,53 @@ $(document).ready(function(){
  	var randomSlide2 = images2[Math.floor(Math.random() * images.length)];
  	$('#slide2').addClass(randomSlide2);
 
+        if ($("#descriptions_display").length){
```

Comment:
I feel we can take advice from @mbarnett re: javascript - putting the code into a separate javascript and not include it in application.js, and only load the javascript when needed. I can work on this once I confirm the rest of the PR works properly.  Are you okay with that @criedlberger ?